src/Navigation/Skill/LanguageItems.js

```diff
@@ -0,0 +1,21 @@
+import React from "react";
+import data from "../../resume.json";
+import "./index.css";
+const skillImage = require.context("../../images/");
+function LanguageItems() {
+  return data.languages.map(subLanguages => {
+    return subLanguages.map(({ language, fluency, logo }) => {
+      const imgSource = skillImage(`./${logo}`);
+      return (
+        <>
+          <div className="skill_language_items">
+            <img className="skill_images" src={imgSource} alt="pic" />
```

Comment:
As you know `alt` is for accessibility, for people who can't see, website-reader will read the content of `alt` to provide the context of the image. A good developer would provide a more precise value of alt.